### ISSUE TITLE
[PREEMPTIVE] - We recently implemented changes in pull job `linux-jammy-py3.8-gcc11 / build` 

### ISSUE THREAD

**Issue by jeanschmidt** [pytorch/pytorch#122604]
TLDR: We are merging [a PR](https://github.com/pytorch/pytorch/pull/122503) on main, changing workflows files for `linux-jammy-py3.8-gcc11 / build`. This might require a rebase for some PRs to properly run `pull.yml` jobs.

## Current Status
* preemptive

## Error looks like
* Something related to file `./.github/workflows/_linux-build-rg.yml` not found;
* Something related to custom action `.github/actions/linux-build/action.yml` not found;
```
Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/ec2-user/actions-runner/_work/pytorch/pytorch/.github/actions/linux-build'. Did you forget to run actions/checkout before running your local action?
```

## Incident timeline 
* Mar 26 - 07:45 PDT [15:45 CET] - [Changes merged](https://github.com/pytorch/pytorch/pull/122503) into production;

## User impact
Due to the [nature of how Github CI runs](https://github.com/pytorch/pytorch/blob/main/CONTRIBUTING.md#which-commit-is-used-in-ci), a rebase might be required for pull.yml jobs to properly start in some PRs where the base is older than https://github.com/pytorch/pytorch/commit/821ad56ea666099c7374914d6f42a6bd757823b8;

## Root cause
The root cause for this issue is related to the fact that the main `trunk.yml` workflow that runs in your PR is based on the merged version of trunk and your branch (so contain main changes), but all the other files and reusable actions and workflows comes strictly from your branch. When there is a dependency between them, they need to be updated to the latest version.

cc @seemethere @malfet @pytorch/pytorch-dev-infra

**Comment by jeanschmidt** [comment]
Delaying the implementation of the change, as a landrace have been identified in the mentioned PR

**Comment by malfet** [comment]
Closing?
